openvm/tests/apc_builder_outputs/load_bytes.txt

```diff
@@ -0,0 +1,98 @@
+mult=is_valid * 1, args=[mem_ptr_limbs__1_0, 13]
+mult=is_valid * 1, args=[read_data_aux__base__timestamp_lt_aux__lower_decomp__0_0, 17]
+mult=is_valid * 1, args=[read_data_aux__base__timestamp_lt_aux__lower_decomp__1_0, 12]
+mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__0_0, 17]
+mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__1_0, 12]
+mult=is_valid * 1, args=[shifted_read_data__0_1 * (1 - opcode_loadb_flag1_1) + shifted_read_data__1_1 * opcode_loadb_flag1_1 - 128 * data_most_sig_bit_1, 7]
+mult=is_valid * 1, args=[503316480 * opcode_loadb_flag1_1 + 1006632960 * shift_most_sig_bit_1 - 503316480 * mem_ptr_limbs__0_1, 14]
+mult=is_valid * 1, args=[mem_ptr_limbs__1_1, 13]
+mult=is_valid * 1, args=[read_data_aux__base__timestamp_lt_aux__lower_decomp__0_1, 17]
+mult=is_valid * 1, args=[read_data_aux__base__timestamp_lt_aux__lower_decomp__1_1, 12]
+mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__0_1, 17]
+mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__1_1, 12]
+mult=is_valid * 1, args=[shifted_read_data__0_2 * (1 - opcode_loadb_flag1_2) + shifted_read_data__1_2 * opcode_loadb_flag1_2 + 773725726 * writes_aux__prev_data__3_8, 7]
+mult=is_valid * 1, args=[503316480 * opcode_loadb_flag1_2 + 1006632960 * shift_most_sig_bit_2 - 503316480 * mem_ptr_limbs__0_2, 14]
+mult=is_valid * 1, args=[mem_ptr_limbs__1_2, 13]
+mult=is_valid * 1, args=[read_data_aux__base__timestamp_lt_aux__lower_decomp__0_2, 17]
+mult=is_valid * 1, args=[read_data_aux__base__timestamp_lt_aux__lower_decomp__1_2, 12]
+mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__0_2, 17]
+mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__1_2, 12]
+mult=is_valid * 1, args=[shifted_read_data__0_3 * (1 - opcode_loadb_flag1_3) + shifted_read_data__1_3 * opcode_loadb_flag1_3 + 773725726 * b__3_7, 7]
+mult=is_valid * 1, args=[503316480 * opcode_loadb_flag1_3 + 1006632960 * shift_most_sig_bit_3 - 503316480 * mem_ptr_limbs__0_3, 14]
+mult=is_valid * 1, args=[mem_ptr_limbs__1_3, 13]
+mult=is_valid * 1, args=[read_data_aux__base__timestamp_lt_aux__lower_decomp__0_3, 17]
+mult=is_valid * 1, args=[read_data_aux__base__timestamp_lt_aux__lower_decomp__1_3, 12]
+mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__0_3, 17]
+mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__1_3, 12]
+mult=is_valid * 1, args=[-(503316480 * mem_ptr_limbs__0_10), 14]
+mult=is_valid * 1, args=[mem_ptr_limbs__1_10, 13]
+mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__0_10, 17]
+mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__1_10, 12]
+
+// Bus 6 (BITWISE_LOOKUP):
+mult=is_valid * 1, args=[(1 - opcode_loadb_flag1_0) * shifted_read_data__0_0 + opcode_loadb_flag1_0 * shifted_read_data__1_0, writes_aux__prev_data__3_9, 0, 0]
```

Comment:
If my code is correct, then it really seems to optimize away the OR opcodes. Nice!

Reply:
Although I don't understand why and how it does it, to be honest.